Script.md

```diff
@@ -0,0 +1,11 @@
+## Introducing Lodgel – the property management application that streamlines the rental process.

```

Comment:
Please add an elevator pitch to this, which should open up the video

Reply:
Added 😁